direct/nn/vsharp/config.py

```diff
@@ -34,3 +34,20 @@ class VSharpNetConfig(ModelConfig):
     image_conv_n_convs: int = 15
     image_conv_activation: str = ActivationType.RELU
     image_conv_batchnorm: bool = False
+
+
+@dataclass
+class VSharpNet3DConfig(ModelConfig):
+    num_steps: int = 8
+    num_steps_dc_gd: int = 6
+    image_init: InitType = InitType.SENSE
+    no_parameter_sharing: bool = True
+    auxiliary_steps: int = -1
+    initializer_channels: tuple[int, ...] = (32, 32, 64, 64)
+    initializer_dilations: tuple[int, ...] = (1, 1, 2, 4)
+    initializer_multiscale: int = 1
```

Comment:
Can't this be tuple[int, int, int, int]?

Reply:
It's not necessarily a sequence of 4 ints.